packages/connectors/react/misc/test-suite.js

```diff
@@ -10,11 +10,7 @@ import { mount } from 'enzyme'
 import { createStore } from 'k-ramel'
 
 export default (lib) => {
-  const {
-    provider,
-    inject,
-    listen,
-  } = lib
+  const { provider, inject, listen } = lib
```

Comment:
Can you desactivate your prettier ?

Reply:
It's already desactivated. This is caused by `eslint --fix` I think... It seems that the test files are not linted by the CI